tests/sqllogictests/suites/base/05_ddl/05_0030_ddl_alter_table_update

```diff
@@ -0,0 +1,46 @@
+statement ok
```

Comment:
Better to move to `0_stateless/17_altertable/17_0003_alter_table_update`?

Reply:
removed, cause this test case is duplicated.